ssas/service/public/api.go

```diff
@@ -69,7 +70,7 @@ func ResetSecret(w http.ResponseWriter, r *http.Request) {
 	setHeaders(w)
 
 	if rd, err = readRegData(r); err != nil || rd.GroupID == "" {
-		service.GetLogEntry(r).Println("missing or invalid GroupID")
+		log.GetCtxLogger(r.Context()).Println("missing or invalid GroupID")
```

Comment:
For these error handling sections that contain a log entry and `service.JSONError`, there will be two logs generated, since `JSONError` [also logs an event](https://github.com/CMSgov/bcda-ssas-app/blob/master/ssas/service/api_common.go#L34).

We could pass along a string into the JSONError function (missing or invalid, etc) + the ctx, or remove the log statement from the function and have the error or info logged out here in the error handling.